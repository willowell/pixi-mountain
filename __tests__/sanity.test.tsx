import renderer from 'react-test-renderer'

describe('Header', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<h1>Hello World!</h1>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
