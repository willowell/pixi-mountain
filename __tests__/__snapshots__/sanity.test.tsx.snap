// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<h1>
  Hello World!
</h1>
`;
